test/functional/save_pages_controller_test.rb

```diff
@@ -0,0 +1,42 @@
+require "test_helper"
+require "gds_api/test_helpers/account_api"
+require "pry"
+
+class SavePagesControllerTest < ActionController::TestCase
```

Comment:
I tried adding some integration tests too, but they ended up doing pretty much the same thing.
So either i need to broaden what I'm testing there or I think this stuff will do?, it's a fairly simple controller for now